scripts/install-mender.sh

```diff
@@ -202,19 +202,29 @@ get_deps() {
 add_repo() {
     curl -fsSL $REPO_URL/gpg | apt-key add -
 
-    repo="deb [arch=$ARCH] $REPO_URL $CHANNEL main"
-
-    echo "Checking if mender sources already exist in '/etc/apt/sources.list'..."
-    if grep -F "$repo" /etc/apt/sources.list; then
-        echo "Removing the old mender debian source list from /etc/apt/sources.list..."
-        if ! sed -i.bak -e "\,$REPO_URL,d" /etc/apt/sources.list; then
-            echo "Failed to remove the existing mender debian source from '/etc/apt/sources.list'."
-            echo "This probably means that there already exists a source in your sources.list."
-            echo "Please remove it manually before proceeding."
-            exit 1
-        fi
+    repo_deprecated="deb [arch=$ARCH] $REPO_URL $CHANNEL main"
+    if grep -F "$repo_deprecated" /etc/apt/sources.list; then
+        echo "ERROR: deprecated repository found in apt sources lists."
+        echo "Please remove it manually with: sudo sed -i.bak -e \"\,$repo_deprecated,d\" /etc/apt/sources.list"
+        echo "See https://docs.mender.io for updated APT repos information"
+        exit 1
```

Comment:
are we sure about exits here? or should be returns?

Reply:
We should exit because if we don't install the repo we just cannot install the packages...